### PR TITLE
Install pip packages with venv

### PR DESCRIPTION
**Pull request by salmanulfarzy**
Manage python dependencies with venv

### CODE REVIEW

**Comment by sadu** [comment]
duplicates #23

File: README.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ cp .env.example .env
 Install dependencies
 
 ```
-pip3 install -r requirements.txt
+python3 -m venv .env
+source .env/bin/activate 
+pip install -r requirements.txt
 ```
```
Comment (joicemjoseph) [inline README.md]: pip3 is needed, right ??
better to use both `virtualenv` and `virtualenvwrapper`

Comment (sidharthv96) [reply README.md]: pipenv solves the issue.

Comment (salmanulfarzy) [reply README.md]: Since venv was created with `python3`, `pip` inside `venv` would suffice.

Comment (salmanulfarzy) [reply README.md]: > pipenv solves the issue.

Setting up `pipenv` on different OSes has multiple issues, It's better to stick to basics for quick setup.

Comment (hemanth7787) [reply README.md]: better to include a pip update option before installing requirements,  without this pip results in error in some OSes. 
`pip install --upgrade pip`

Comment (salmanulfarzy) [reply README.md]: There were some migration issue on `pip 9` to `pip 10`, It's better to leave `pip` version management to the user.

**Comment by jilvin** [comment]
@salmanulfarzy Duplicates #23. Please review #23 and contribute to the same.

**Comment by remedcu** [comment]
You may close this PR.

Related ones: #23 #26 #33 #392 

I think just #392 (if required) is enough.

@syam3526 @vigneshhari @biswaz @vrthra

**Pull request closed without merging**